.travis.yml

```diff
@@ -4,7 +4,6 @@ python:
   - 3.3
   - 3.4
   - 3.5
-  - 3.6
```

Comment:
Thanks for this suggestion, I've tried it and it worked. Although it will be not correct in case they change the nightly build to 3.7 :wink: It is on the main branch now.